docs/subnets/register-validate-mine.md

```diff
@@ -192,3 +192,61 @@ After providing your wallet name at the prompt, you will see the following outpu
 
 
 In the above table, the `ACTIVE` row is applicable only for UIDs that are subnet validators. It shows whether the UID is actively setting the weights within the [`activity_cutoff`](./subnet-hyperparameters#activity_cutoff) window. If the UID has not set weights on the blockchain for `activity_cutoff` duration, then the Yuma Consensus will consider this subnet validator as offline, i.e., turned off (`False`).
+
+## Checking the registration status
+
+To check the registration status of your UID, use any one of the below Python code fragments. To use these code fragments:
+
+- **Using Python interpreter**: A straightforward way is to type "python" or "python3" on your macOS or Linux terminal and it will open up a Python interpreter. Copy and paste the entire code fragment into it and hit "Return" and see the output.
+- **Using Python compiler**: Or you can use the Python compiler. Copy the code fragment into a file, say, "check_reg.py" and run the command `python3 check_reg.py` or `python check_reg.py` on your terminal command line and see the output.
```

Comment:
It may be appropriate to use `binary` here.  Python is not a compiled language, but when running a python script, you'd be using the python binary this way.

~~compiler~~ -> binary